src/sensors/src/Accelerometer.cpp

```diff
@@ -139,6 +138,20 @@ SensorType Accelerometer::getSensorType() const
     return ACCELEROMETER;
 }
 
+bool Accelerometer::getLinkSensorTransform(Transform& link_H_sensor) const
+{
+    if( this->pimpl->parent_link_index < 0)
```

Comment:
If the code needs that information, it will call the getter for the parent_link_index, and it will get the invalid value for it.